cmd/create/service/cmd.go

```diff
@@ -0,0 +1,360 @@
+/*
+Copyright (c) 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package service
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/spf13/cobra"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/rosa/pkg/arguments"
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/info"
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/properties"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+)
+
+var args ocm.CreateManagedServiceArgs
+
+var Cmd = &cobra.Command{
+	Use:     "service",
+	Aliases: []string{"appliance"},
+	Short:   "Creates a managed service.",
+	Long: `  Managed Services are OpenShift clusters that provide a specific function.
```

Comment:
This command is not showing up in the help 
```
rosa create --help 
Create a resource from stdin

Usage:
  rosa create [command]

Aliases:
  create, add

Available Commands:
  account-roles  Create account-wide IAM roles before creating your cluster.
  admin          Creates an admin user to login to the cluster
  cluster        Create cluster
  idp            Add IDP for cluster
  ingress        Add Ingress to cluster
  machinepool    Add machine pool to cluster
  oidc-provider  Create OIDC provider for an STS cluster.
  operator-roles Create operator IAM roles for a cluster.

Flags:
  -h, --help             help for create
      --profile string   Use a specific AWS profile from your credential file.
      --region string    Use a specific AWS region, overriding the AWS_REGION environment variable.
  -y, --yes              Automatically answer yes to confirm operation.

Global Flags:
      --debug   Enable debug mode.

Use "rosa create [command] --help" for more information about a command.

```

Reply:
That's on purpose: it's a hidden command (see `Hidden: true` a few lines below)

Reply:
Resolve?